src/main/scala/mesosphere/marathon/core/volume/Volumes.scala

```diff
@@ -0,0 +1,97 @@
+package mesosphere.marathon.core.volume
```

Comment:
All packages under core are components that contain:
- interface definitions (traits)
- impl that holds private implementation
- Module that defines the required/provided interfaces 

We are moving the code from `*` to `core/module` thats why it is probably confusing.
But moving Volumes here feels wrong.